demo/aws/aws-tomcat/configure-database.tf

```diff
@@ -0,0 +1,102 @@
+resource "aws_security_group" "ubuntu-mysql" {
+  name        = "ubuntu-mysql-security-group"
```

Comment:
Replace prefix with `test-..` or `sample-...` for clarity, in case if someone forgets to destroy these resources, so we can trace them.

Reply:
done